libs/vm-modules/tests/unit/ml/ml_model_tests.cpp

```diff
@@ -1137,6 +1165,46 @@ TEST_F(VMModelTests, model_sequential_flatten_2d_in_2d_out)
   EXPECT_TRUE(constructed_shape == expected.shape());
 }
 
+TEST_F(VMModelTests, model_sequential_reshape_3d_in_2d_out)
```

Comment:
isn't this 1d in and 2d out?

Reply:
nope :) when you call tensor.fromString() method it reinits the tensor so it doesnt matter how to init an initial shape. it could be even {1,2,3,4,5,6,7,8,9} shape. BTW we have a task about binding fromString constructor to avoid these unpleasant initial array creation. In fact, a tensor with shape {2,3,1} is created in this test